molecule/command/init/collection.py

```diff
@@ -0,0 +1,191 @@
+            # collection_name,
+        )
+        templates = [
+            'scenario/driver/{driver_name}'.format(**self._command_args),
+            'scenario/verifier/{verifier_name}'.format(**self._command_args),
+        ]
+        for template in templates:
+            self._process_templates(template, self._command_args,
+                                    scenario_base_directory)
+        self._process_templates(
+            'molecule', self._command_args, collection_directory,
+        )
+
+        msg = 'Initialized colletion in {} successfully.'.format(
+            collection_path)
+
+        LOG.success(msg)
+
+
+@click.command()
+@click.pass_context
+@click.option(
+    '--dependency-name',
+    type=click.Choice(['galaxy']),
+    default='galaxy',
+    help='Name of dependency to initialize. (galaxy)')
+@click.option(
+    '--driver-name',
+    '-d',
+    type=click.Choice(config.molecule_drivers()),
+    default='docker',
+    help='Name of driver to initialize. (docker)')
+@click.option(
+    '--lint-name',
+    type=click.Choice(['yamllint']),
+    default='yamllint',
+    help='Name of lint to initialize. (yamllint)')
+@click.option(
+    '--collection-namespace', '-n', required=True,
+    help='Namespace of the collection.',
+)
+@click.option(
+    '--collection-name', '-c', required=True,
+    help='Name of the collection.',
+)
+@click.option(
+    '--directory',
+    '-d',
+    required=False,
+    help='Directory path for the new collection.')
+@click.option(
+    '--provisioner-name',
+    type=click.Choice(['ansible']),
+    default='ansible',
+    help='Name of provisioner to initialize. (ansible)')
+@click.option(
+    '--template',
+    '-t',
+    type=click.Path(
+        exists=True, dir_okay=True, readable=True, resolve_path=True),
+    help='Path to a cookiecutter custom template to initialize the '
+    'collection. The upstream molecule folder will be added to '
+    'this template.',
+)
+@click.option(
+    '--verifier-name',
+    type=click.Choice(config.molecule_verifiers()),
+    default='testinfra',
+    help='Name of verifier to initialize. (testinfra)')
+def collection(
+        ctx,  # pylint: disable=unused-argument
+        dependency_name,
+        directory, driver_name,
+        collection_namespace, collection_name,
+        lint_name, provisioner_name,
+        template, verifier_name,
+):  # pragma: no cover
+    """Initialize a new collection for use with Molecule."""
+    command_args = {
+        'collection_namespace': collection_namespace,
+        'collection_name': collection_name,
+        'collection_directory': directory,
+        'dependency_name': dependency_name,
+        'driver_name': driver_name,
+        'lint_name': lint_name,
+        'provisioner_name': provisioner_name,
+        'scenario_name': command_base.MOLECULE_DEFAULT_SCENARIO_NAME,
+        'subcommand': __name__,
+        'verifier_name': verifier_name,
+    }
+
+    verifier_linter_map = {
```

Comment:
Is testinfra missing here on purpose?

Reply:
It's copied from role init and collapsed into a dict. I didn't invent anything different...